app/__init__.py

```diff
@@ -37,6 +37,12 @@ def create_app():
             '/applications/<string:hash_id>/status',
         )
 
+        api.add_resource(
+            routes.Attachments,
+            '/attachments',
+            '/attachments/<string:hash_id>/attachment/<string:attachment_id>',
+        )
+
```

Comment:
Multiple endpoints defined on single resource, this behavior is not recommended from the python restful library docs. The library docs recommends that each endpoint has it's own resource.

```
add.api_resource(routes.AttachmentList, '/attachments')
add.api_resource(routes.Attachment, '/attachments/<string:hash_id>/attachment/<string:attachment_id>')
```

Also unclear if the retrieval of a singel attachment is necessary?

Reply:
Fixed